src/assets/style/variables/resolution.scss

```diff
@@ -1,12 +1,12 @@
 $breakpoints: (
   "mobile": (
-    min-width: 375px
+    max-width: 375px
   ),
   "tablet": (
-    min-width: 768px
+    max-width: 768px
   ),
   "desktop": (
-    min-width: 1200px
+    min-width: 769px
```

Comment:
Are you sure that desktop view should start from 769px instead of 1200?

Reply:
My mistake.